ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -300,4 +303,9 @@ create_catalog_sources
 # And, there is no brew.registry.redhat.io/redhat/redhat-operator-index:v4.11 , so disable the default CatalogSources.
 # TODO: the Proxy registry support the `registry.redhat.io` images
 # check_default_catalog
-disable_default_catalogsource
+
+# No need to disable the default OperatorHub when marketplace disabled as default.
+if [ $marketplace -eq 0]; then
```

Comment:
missing an extra whitespace, should be `[ $marketplace -eq 0 ]`

Reply:
Great catch! Thanks!